modules/ckanext-ytp_main/ckanext/ytp/schemas/dataset.json

```diff
@@ -408,6 +408,21 @@
       "field_name": "temporal_coverage_from",
       "label": "Temporal coverage from",
       "preset": "date_opendata"
+    },
+    {
+      "field_name": "status_updated",
+      "form_snippet": "hidden.html",
+      "validators": "ignore_missing admin_only_field"
+    },
+    {
+      "field_name": "sha256",
+      "form_snippet": "hidden.html",
+      "validators": "ignore_missing admin_only_field"
+    },
+    {
+      "field_name": "malware_check",
+      "form_snippet": "hidden.html",
```

Comment:
Is the purpose to keep them hidden on the page or just not display them at all? I think if you pass null to display_snippet you can remove a field from being displayed and being posted at all. (https://github.com/ckan/ckanext-scheming#display_snippet).

Or is the purpose to keep it and just hide it?

Reply:
Yeah, that actually needs a bit more work. SHA256 and malware check result should show up on the page, but the `status_updated` not necessarily. I'll focus on the UI side more once the action part works.

Reply:
Show up on the view page, but not in the edit view, I mean.